code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/AuthDao.java

```diff
@@ -18,7 +22,29 @@
 
     Account createAccount(String name, String kind, String externalId, String externalType);
 
+    Account createProject(String name, String kind);
```

Comment:
What would be the point? we have no other project Types. Maybe in the future if we have more project types.